### ISSUE TITLE
[Lighthouse a11y] Vague error suggestion for "month" and "day" form fields

### ISSUE THREAD

**Issue by shreyachatterjee00** [civiform/civiform#12890]
**Describe the bug**
Issue: Vague Error Suggestion for "Month" and "Day" Form Fields. The text error suggestion "Please enter a date in the correct format"does not indicate which Form Field has an invalid entry and how to remediate it. 

Location: Comprehensive Sample Program Form Screen 1

**WCAG Success Criterion** 
3.3.3 Error Suggestion

**To Reproduce**

1. Properly fill in 1 or 2 of the Birthday group Form Fields Month, Day, and Year while leaving 1 or 2 Form Fields blank.
 2. Select the "Continue" Button.
 
 Or
 
 1. Properly fill the "Month" and "Year" Form Field.
 2. Input an invalid entry for "Day" e.g. 32.
 3. Select the "Continue" Button.

**Expected behavior**
  Expectation: A clear and logical error suggestion should be announced.

**Remediation**
Provide a text error suggestion that is clear and logical.
 E.g. Error "Month" is required, Error "Date" Enter a digit between 1-31
